levels/fullstack.tex

```diff
@@ -0,0 +1,251 @@
+\documentclass{article}
+
+\usepackage[table]{xcolor}
+\usepackage[paperwidth=800mm, paperheight=500mm]{geometry}
+
+% Don't show page numbers.
+\pagenumbering{gobble}
+
+% Column settings for skills able.
+\newcolumntype{C}{p{18em}}
+
+\begin{document}
+
+% Title of matrix.
+\begin{center}
+\begin{huge}
+Fullstack Engineering
```

Comment:
This is for fullstack but reads to me like a more advanced front-end criteria. 

I was thinking that in lvl 3 and above we should add a backend focused element like...
- Can author a basic webserver without frameworks using a serverside language (go, node, etc)